### DISCUSSION TITLE
[缺陷] 缺少获取用户头像的API

### DISCUSSION THREAD

**Discussion by ZeroAurora** [howmanybots/onebot#73] [想法 / 建议]
在全篇公开 API 的描述中，唯一一个与头像有关的 API 是 [get_group_honor_info](https://github.com/howmanybots/onebot/blob/master/v11/specs/api/public.md#get_group_honor_info-%E8%8E%B7%E5%8F%96%E7%BE%A4%E8%8D%A3%E8%AA%89%E4%BF%A1%E6%81%AF) 中。私以为这是设计的一个失误。
这些 API 应该返回用户头像：get_stranger_info, get_group_member_info
这些 event 应该返回用户头像：消息事件.sender

**Comment by Mrs4s** [comment]
QQ的头像可以直接通过拼接URL来获取: `http://q1.qlogo.cn/g?b=qq&nk={id}&s=640`
不过考虑到跨平台确实可以单独定义个API来获取头像

**Comment by ExerciseBook** [comment]
@richardchien 这个问题也加入 V12 里吧

**Comment by acaly** [comment]
想请问一下头像相关的问题：

现在有没有类似方法获取群头像呢？
有没有可能加入设置头像的接口（群、个人）？

**Comment by stdrc** [comment]
> @richardchien 这个问题也加入 V12 里吧

可以，大家有兴趣都可以 PR～
